src/services/expenses.services.js

```diff
@@ -0,0 +1,47 @@
+const { getExpensesFilterQuery } = require('../utils/expenses');
+
+const { Expense } = require('../models/Expense.model');
+
+const getExpenses = async (categories, userId, from, to) => {
+  const where = getExpensesFilterQuery(categories, userId, from, to);
```

Comment:
Ensure that the function name `getExpensesFilterQuery` is correctly spelled and consistent across the codebase. Previously, there was a typo in the function name in another file.